AcadeMeData/tests/test_messages.py

```diff
@@ -0,0 +1,30 @@
+import pytest
+from AcadeMeData.models import Messages, User
+
+
+@pytest.mark.django_db
+class TestMessagesModel:
+    @pytest.fixture
+    def user_example(self):
+        user_data = {'username': "username2212", 'password': "password", 'email': "user@example.com", 'type': "S",
+                     'university': "RU",
+                     'degree': "CS"}
+        user = User.create_user(*user_data)
+        return user
+
+    def generate_message(self, msgID=1, userID=None, text='This is a test message yo'):
+        message = Messages(msgID=msgID, userID=userID, text=text)
+        message.save()
+        return message
+
+    def test_get_msg(self, user_example, msgID=1, userID=None, text='This is a test message yo'):
```

Comment:
Test should call a fixture, 
And then just search in object with a search filter and assert.
Please take a look at test #60